plugins/woocommerce/tests/e2e-pw/bin/test-env-setup.sh

```diff
@@ -17,6 +17,9 @@ wp-env run tests-cli wp plugin activate process-waiting-actions
 echo -e 'Activate Test Helper APIs utility plugin \n'
 wp-env run tests-cli wp plugin activate test-helper-apis
 
+echo -e 'Install Plugin-check utility plugin \n'
+wp-env run tests-cli wp plugin install plugin-check --activate
```

Comment:
We may want to think about putting this in a separate [env context](https://github.com/woocommerce/woocommerce/tree/9a58378fec1b21c9d8c3f838f5089d0a92885881/plugins/woocommerce/tests/e2e-pw/envs), so that we're not installing the plugin for every e2e run. Not a big thing, but I think it would be cleaner and more consistent.

Reply:
Agree + downloading the plugin to add extra execution time and additional points of failure.